Python/pytime.c

```diff
@@ -1050,26 +1050,14 @@ py_win_perf_counter_frequency(LONGLONG *pfrequency, int raise)
     LONGLONG frequency;
 
     LARGE_INTEGER freq;
-    if (!QueryPerformanceFrequency(&freq)) {
-        if (raise) {
-            PyErr_SetFromWindowsErr(0);
-        }
-        return -1;
-    }
+    // Since Windows XP, the function cannot fail.
+    (void)QueryPerformanceFrequency(&freq);
```

Comment:
Why query this every time? It's fixed at boot, so it isn't going to change.

Reply:
This function must be called exactly once in a process: py_get_win_perf_counter() uses a static variable for that.